perception/map_based_prediction/README.md

```diff
@@ -0,0 +1,64 @@
+
+3. Confidence calculation
+   We use the following metric to compute the distance to a certain lane.
+
+   ```txt
+   d = x^T P x
+   ```
+
+   where `x=[lateral_dist, yaw_diff]` and `P` are covariance matrices. Therefore confidence values can be computed as
+
+   ```txt
+   confidence = 1/d
+   ```
+
+   Finally, we normalize the confidence value to make it as probability value. Note that the standard deviation of the lateral distance and yaw difference is given by the user.
+
+4. Drawing predicted trajectories
+   From the current position and reference trajectories that we get in the step1, we create predicted trajectories by using Quintic polynomial. Note that, since this algorithm consider lateral and longitudinal motions separately, it sometimes generates dynamically-infeasible trajectories when the vehicle travels at a low speed. To deal with this problem, we only make straight line predictions when the vehicle speed is lower than a certain value (which is given as a parameter).
+
+## Inputs / Outputs
+
+### Input
+
+| Name                                               | Type                                                 | Description                              |
+| -------------------------------------------------- | ---------------------------------------------------- | ---------------------------------------- |
+| `~/perception/object_recognition/tracking/objects` | `autoware_auto_perception_msgs::msg::TrackedObjects` | tracking objects without predicted path. |
+| `~/vector_map`                                     | `autoware_auto_mapping_msgs::msg::HADMapBin`         | binary data of Lanelet2 Map.             |
+
+### Output
+
+| Name                     | Type                                                   | Description                           |
+| ------------------------ | ------------------------------------------------------ | ------------------------------------- |
+| `~/objects`              | `autoware_auto_perception_msgs::msg::PredictedObjects` | tracking objects with predicted path. |
+| `~/objects_path_markers` | `visualization_msgs::msg::MarkerArray`                 | marker for visualization.             |
+
+## Parameters
+
+| Parameter                        | Type   | Description                                |
+| -------------------------------- | ------ | ------------------------------------------ |
+| `has_subscribed_map`             | bool   | true when subscribed path exists           |
+| `prediction_time_horizon`        | double | predict time duration for predicted path   |
+| `prediction_sampling_delta_time` | double | sampling time for points in predicted path |
```

Comment:
Many parameters are missing.
Could you merge this PR after it is merged to AutowareArchitectureProposal.iv?

https://github.com/tier4/AutowareArchitectureProposal.iv/pull/762

Reply:
I merged it https://github.com/autowarefoundation/autoware.universe/pull/69/commits/b35f7c586fb17b9284f010b3c5a5b0e6c2c2012b